crates/aptos/src/governance/utils.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{governance::*, *};
+use aptos_types::on_chain_config::FeatureFlag;
+
+pub fn parse_vote_option(yes: bool, no: bool) -> CliTypedResult<bool> {
```

Comment:
Instead of creating a util func like this, I think it would be better to add this func in `SubmitVote`, that way you can grab the Yes/No param directly instead of passing in.